src/api/axiosInstance.js

```diff
@@ -17,9 +17,13 @@ import $ from 'jquery';
  *
  * @module Axios
  */
-const createRequestInstance = (resolve) => {
+const createRequestInstance = (resolve, timeout) => {
+  if (timeout === undefined) {
+    timeout = 10000;
```

Comment:
Maybe it should be a parameter in the config file.

Reply:
Done